data/roleConfig.txt

```diff
@@ -0,0 +1,8 @@
+USAGE: config <add|remove> <Role Name> 
```

Comment:
We're getting to a point now where these help texts should be put into their own folder I think